api/views.py

```diff
@@ -31,3 +33,20 @@ def get_object(self):
         queryset = Item.objects.filter(assigned_to=self.request.user)
         obj = get_object_or_404(queryset, serial_number=self.kwargs['pk'])
         return obj
+
+
+class SecurityUserViewSet(ModelViewSet):
+    serializer_class = SecuritySerializer
+    permission_classes = [IsAuthenticated, ]
+    authentication_classes = (FirebaseTokenAuthentication,)
```

Comment:
as per our discussion, let's have a different authentication class here.

Reply:
Okay, then should this ticket still include writing how the authentication should work? I think that should be a ticket on its own. What do you think ?